.circleci/config.yml

```diff
@@ -19,25 +19,25 @@ jobs:
 
   publish_image_to_quay:
     docker:
-      - image: circleci/openjdk:8-jdk-stretch
+      - image: docker.io/docker:20.10-git
```

Comment:
I don't think we should be adding dependencies on docker hub given the rate limits.

Reply:
True. Although I doubt it would be a problem.